builtin/logical/pki/path_revoke.go

```diff
@@ -151,6 +156,11 @@ signed by an issuer in this mount.`,
 				Description: `Key to use to verify revocation permission; must
 be in PEM format.`,
 			},
+			"allow_expired_cert_revocation": {
```

Comment:
I think this should go on `config/crl` (in `path_config_crl`), not `revoke`: we want this to be a conscious choice by the operator to enable, globally, and not a per-certificate action by someone potentially unrelated. This is because it requires extra bookkeeping and cost and so shouldn't be done arbitrarily. My 2c! :-)

Reply:
Oh yea, that makes sense, I didn't look at it that way. I'll make the changes :)

Reply:
@cipherboy I made the changes and created a test, I'll work on `revoked_safety_buffer` now :)

Reply:
Nice! That one goes on the tidy operation, not the crl config, btw :-)

Reply:
Alrighty!